wrangler.toml

```diff
@@ -3,19 +3,24 @@ main = "src/worker.ts"
 compatibility_date = "2023-07-17"
 
 
-
 [env.staging]
 kv_namespaces = [
   { binding = "CronJobsTimestamp", id = "509cd24483a24ddcb79f85f657274508" }
 ]
+services = [
+  { binding = "DISCORD_BOT", service = "discord-slash-commands", environment = "staging" }
+]
 
 [env.production]
 kv_namespaces = [
 	{ binding = "CronJobsTimestamp", id = "3a10f726c95d4afea9dee5fd00f029b9" }
 ]
+services = [
+  { binding = "DISCORD_BOT", service = "discord-slash-commands", environment = "production" }
+]
 
 [triggers]
-crons = ["0 */4 * * *", "0 */6 * * *"]
+crons = ["0 */4 * * *", "0 */6 * * *","0 */12 * * *" ]
```

Comment:
didn't get this line. can you please explain what is being updated here

Reply:
Adds a cron tab entry for every 12hr